templates/components/products/description-tabs.html

```diff
@@ -7,6 +7,11 @@
             <a class="tab-title" href="#tab-warranty">{{lang 'products.warranty'}}</a>
         </li>
     {{/if}}
+    {{#all product.custom_fields theme_settings.show_custom_fields_tabs}}
+        <li class="tab">
+            <a class="tab-title" href="#tab-specs">{{theme_settings.pdp-custom-fields-tab-label}}</a>
```

Comment:
Can we build the href based on the label so it matches what we are actually calling the tab.

Reply:
<img width="1552" alt="Screen Shot 2019-04-22 at 4 23 39 PM" src="https://user-images.githubusercontent.com/5056945/56539748-a21f6680-651b-11e9-9d57-37e5dacfedbd.png">